application/src/main/java/dev/stratospheric/todoapp/todo/TodoController.java

```diff
@@ -35,7 +35,7 @@ public String showView(
     Model model
   ) {
 
-    Todo todo = todoService.getOwnedTodo(id, user.getEmail());
+    Todo todo = todoService.getOwnedOrShared(id, user.getEmail());
```

Comment:
This method should probably be named `getOwnedOrSharedTodo`

Reply:
🦅 eye - you're right, thanks